starlette/datastructures.py

```diff
@@ -254,7 +261,7 @@ def __init__(
         if kwargs:
             value = (
                 ImmutableMultiDict(value).multi_items()
-                + ImmutableMultiDict(kwargs).multi_items()
+                + ImmutableMultiDict(kwargs).multi_items()  # type: ignore[operator]
```

Comment:
We don't know the returned value of the operator `+` when there's a sum of two `ImmutableMultiDict`. That's why we ignore this error here.